ExceptionsHomework/src/ro/scit/tests/MenuTest.java

```diff
@@ -0,0 +1,61 @@
+package ro.scit.tests;
+
+import org.junit.Assert;
+import org.junit.Test;
+import ro.scit.entities.Student;
+import ro.scit.utilities.DateOfBirth;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class MenuTest {
+    List<Student> student = new ArrayList<>();
```

Comment:
If you need to perform some set up before running the actual test methods, you can create a method, say `setUp()`, and annotate it with `@Before` or `@BeforeClass` (depending on your needs).